src/content/doc-integrations/Frameworks/crewai.mdx

```diff
@@ -2,103 +2,84 @@
 sidebar_position: 1
 sidebar_label: CrewAI
 title: CrewAI | Integrations
-description: This section contains information about the CrewAI framework and how to integrate it with SurrealDB.
+description: How to plug SurrealDB into CrewAI as a memory and vector-search layer.
 ---
 
 # CrewAI
 
-[CrewAI](https://github.com/joaomdmoura/crewAI) is a framework for orchestrating role-playing AI agents. It lets you define a "crew" of specialized agents that work together to accomplish complex tasks - like a researcher gathering data that a planner then uses to make recommendations.
+[CrewAI](https://github.com/joaomdmoura/crewAI) lets you orchestrate role-playing AI agents that collaborate to complete complex tasks.  
+This guide shows how to use SurrealDB `>v2.0.0` as the memory backend for CrewAI, giving agents:
 
-This integration shows how to use SurrealDB as the memory layer for CrewAI agents, giving them:
+- **Entity memory** for domain objects (products, people, places)  
+- **Short-term memory** for recent conversations  
+- **Vector search** to recall relevant information by semantic similarity  
 
-- **Entity memory** to store and recall domain objects (products, people, places)
-- **Short-term memory** to reference recent conversations between agents
-- **Vector search** to find relevant past information using semantic similarity
-
-The example below creates two agents that collaborate to recommend music festivals:
-1. A researcher that finds festival candidates and saves them to SurrealDB
-2. A planner that queries those saved festivals to make personalized suggestions
+The sample below creates two agents that recommend music-festival trips:
 
+1. **Researcher** – finds festivals and saves them to SurrealDB  
+2. **Planner** – queries those saved festivals to build a weekend itinerary  
 
 ## Install
 
 ```bash
-# CrewAI, SurrealDB Python SDK, and an embedder (OpenAI here, swap if you like)
+# CrewAI, SurrealDB Python SDK, and an embedder (OpenAI here; swap if you like)
 pip install "crewai[tools]" surrealdb openai
-# (optional) spin up SurrealDB locally – single binary, no deps
+
+# (optional) run SurrealDB locally – single binary, no deps
 docker run --pull always -p 8000:8000 surrealdb/surrealdb:latest \
```

Comment:
Here I think it could be shown as a choice between two commands (docker vs. regular surreal) as opposed to Docker by default and optionally to run locally.

Also I think change file:/data/db to rocksdb://data/db